src/4.3_to_5.0/addLegacyBookmarkToUIFolder.ts

```diff
@@ -0,0 +1,61 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.1";
+import { produce } from "immer";
+
+/**
+ * Returns a new `legacyUIFolder` with the given `legacyBookmarksToAdd` added.
+ * Useful for creating legacyUIFolders for tests.
+ * Does not mutate `legacyUIFolder`.
+ */
+export function addLegacyBookmarkToUIFolder(
```

Comment:
Let's move this function to within `__test_resources__` as it will be used just to generate test resources.

Reply:
Done ✅